src/lcm/random_choice.py

```diff
@@ -3,23 +3,29 @@
 import jax
 
 
-def random_choice(key, probs, labels):
+def random_choice(
+    key: jax.Array,
+    probs: jax.Array,
+    labels: jax.Array,
+) -> jax.Array:
```

Comment:
Add dtype info? Should be ints, right?

Reply:
There is currently no way to specify dtypes of jax arrays without additional packages like jaxtyping.

Reply:
Cool, just add to #107 and adjust the description there.